testing/kuttl/e2e/major-upgrade/02-assert.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: major-upgrade
+spec:
+  postgresVersion: 14
+status:
+  instances:
+    - name: instance1
+      readyReplicas: 1
+      replicas: 1
+      updatedReplicas: 1
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: major-upgrade
+    postgres-operator.crunchydata.com/pgupgrade: ""
+status:
+  succeeded: 1
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: major-upgrade
+    postgres-operator.crunchydata.com/pgbackrest-backup: replica-create
+status:
+  succeeded: 1
```

Comment:
Is this the same replica-create that you tested in 00-assert.yaml? Do we need this check?

Reply:
No, there should be another backup Job after the upgrade, so this is testing a separate step.

Reply:
But the labels here are the same as in 00-assert lines 18-19, so would we see 2 successful pods from this job?

Reply:
There shouldn't be. The upgrade utilizes the shutdown process, so the pods are removed before the upgrade Job is run. In testing, I only saw one backup Pod at a time.